docs/sphinx/source/whatsnew/v0.7.0.rst

```diff
@@ -8,6 +8,7 @@ recommend all users of v0.6.3 upgrade to this release after checking API
 compatibility notes.
 
 **Python 2.7 support ended on June 1, 2019.** (:issue:`501`)
+**Python 2.7 support ended on June 1, 2019**. (:issue:`501`)
```

Comment:
Looks like a duplicate.

Reply:
Good catch, thanks! I put line 10 in to contain the period within the bolding in the rendered version. I guess I didn't get the merge quite right. I'll add this to the PR.